lib/board.rb

```diff
@@ -4,13 +4,13 @@
 class Board
   attr_accessor :cell
   def initialize(current_player)
-    @cell = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    @cell = ['1'.gray, '2'.gray, '3'.gray, '4'.gray, '5'.gray, '6'.gray, '7'.gray, '8'.gray, '9'.gray]
     @current_player = current_player
   end
 
   def update_cell(mark)
-    @cell[mark - 1] = 'X' if @current_player == 'player1'
-    @cell[mark - 1] = 'O' if @current_player == 'player2'
+    @cell[mark - 1] = 'X'.magenta if @current_player == 'player1'
+    @cell[mark - 1] = 'O'.cyan if @current_player == 'player2'
   end
```

Comment:
Instead of testing for the player here, you could have @current_player hold the details of the current player and just pass in @current_player.symbol to the marked cell, somethinglike that